common/lib/xmodule/xmodule/course_module.py

```diff
@@ -176,6 +176,53 @@ class CourseFields(object):
                           help=_('EdEos Secret'))
     edeos_key = String(display_name='EdEos Key', scope=Scope.settings, default="",
                        help=_('EdEos Key, also known as Client Id'))
+    learning_format = String(
+        display_name=_("Learning format"),
+        help=_(
+            'Set the learning format (for example "Intramural", "Online", "Webinars 2 times a week", '
+            '"Video lections" or "Evening online vebinars").'
```

Comment:
Since it's part of GUI, leaving this note:
`vebinars` -> `webinars `

Reply:
Thanks! 👍